python/ray/train/huggingface/huggingface_predictor.py

```diff
@@ -136,35 +137,42 @@ def _predict_pandas(
                 ``pipeline`` object.
 
         Examples:
-
-        .. code-block:: python
-
-            import pandas as pd
-            from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
-            from transformers.pipelines import pipeline
-            from ray.train.huggingface import HuggingFacePredictor
-
-            model_checkpoint = "gpt2"
-            tokenizer_checkpoint = "sgugger/gpt2-like-tokenizer"
-            tokenizer = AutoTokenizer.from_pretrained(tokenizer_checkpoint)
-
-            model_config = AutoConfig.from_pretrained(model_checkpoint)
-            model = AutoModelForCausalLM.from_config(model_config)
-            predictor = HuggingFacePredictor(
-                pipeline=pipeline(
-                    task="text-generation", model=model, tokenizer=tokenizer
-                )
-            )
-
-            prompts = pd.DataFrame(
-                ["Complete me", "And me", "Please complete"], columns=["sentences"]
-            )
-            predictions = predictor.predict(prompts)
+            >>> import pandas as pd
+            >>> from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
+            >>> from transformers.pipelines import pipeline
+            >>> from ray.train.huggingface import HuggingFacePredictor
+            >>>
+            >>> model_checkpoint = "gpt2"
+            >>> tokenizer_checkpoint = "sgugger/gpt2-like-tokenizer"
+            >>> tokenizer = AutoTokenizer.from_pretrained(tokenizer_checkpoint)
+            >>>
+            >>> model_config = AutoConfig.from_pretrained(model_checkpoint)
+            >>> model = AutoModelForCausalLM.from_config(model_config)
+            >>> predictor = HuggingFacePredictor(
+            ...     pipeline=pipeline(
+            ...         task="text-generation", model=model, tokenizer=tokenizer
+            ...     )
+            ... )
+            >>>
+            >>> prompts = pd.DataFrame(
+            ...     ["Complete me", "And me", "Please complete"], columns=["sentences"]
+            ... )
+            >>> predictions = predictor.predict(prompts)
 
 
         Returns:
             Prediction result.
         """
+        return Predictor.predict(
+            self, data, feature_columns=feature_columns, **predict_kwargs
+        )
```

Comment:
@pcmoritz I just noticed this when scanning the code, any particular reason you manually call `Predictor.predict()` rather than using the idiomatic [`super().predict()`](https://docs.python.org/3/library/functions.html#super)?
```python
return super().predict(data, feature_columns=feature_columns, **predict_kwargs)
```

Reply:
I don't think so -- feel free to change it :)